frontend/src/components/adminPage/ProfessionRows.js

```diff
@@ -56,19 +71,28 @@ import '../../style/ProfessionRows.css';
                 || this.state.idDoc!==nextProps.idDoc 
                 || this.props.idProf!==nextProps.idProf 
                 || this.state.idProf!==nextProps.idProf 
-                || 
-                this.state.currentDate!==nextState.currentDate
+                || this.state.currentDate!==nextState.currentDate
                 || this.state.idProf!==nextState.idProf
                 || this.state.idDoc!==nextState.idDoc
-                || this.state.ratesArr!==nextState.ratesArr);
+                || this.state.ratesArr!==nextState.ratesArr
+                || this.state.shouldUpdate!==nextState.shouldUpdate);
```

Comment:
What difference between this record and (this.state !== nextState)?

Reply:
nothing

Reply:
why don't write (this.state !== nextState)?